sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/screens/organisms/InformationMessageCardViewScreen.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.hmrc.sample_compose_fragments.presentation.screens.organisms
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import uk.gov.hmrc.components.compose.organism.HmrcCardView
+import uk.gov.hmrc.components.compose.organism.information.InformationMessageButton
+import uk.gov.hmrc.components.compose.organism.information.InformationMessageButton.Companion.ButtonType
+import uk.gov.hmrc.components.compose.organism.information.InformationMessageCardView
+import uk.gov.hmrc.components.compose.organism.information.MessageType
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme
+import uk.gov.hmrc.sample_compose_components.R
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.ExamplesSlot
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.PlaceholderSlot
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.ScreenScrollViewColumn
+
+@Composable
+fun InformationMessageCardViewScreen(onClickAction: () -> Unit) {
+    ScreenScrollViewColumn {
+        PlaceholderSlot {
+            InformationMessageCardView(
+                headline = stringResource(id = R.string.info_message_placeholder_headline),
+                text = stringResource(id = R.string.info_message_placeholder_headline),
+                messageType = MessageType.URGENT,
+                buttons = listOf(InformationMessageButton("Headline button", MessageType.URGENT, onClick = {}))
+            )
+        }
+
+        ExamplesSlot {
+            // Example 1
+            HmrcCardView(modifier = Modifier.padding(bottom = HmrcTheme.dimensions.hmrcSpacing16)) {
```

Comment:
I don't think we need this card view.. the component has one built in. Same comment applies to the examples below.
```suggestion
```